packages/core/test/spatial-types.test.ts

```diff
@@ -94,14 +94,34 @@ describe('isPoint', () => {
     new Point(CARTESIAN_3D_CRS_CODE, 19.24, 100.29, 20.22222),
     new Point(CARTESIAN_3D_CRS_CODE, 19.24, 100.29),
     new Point(0, 19.24, 100.29, 20.22222)
-  ])('isPoint(%s) should be truthy', point =>
+  ])('isPoint(%s) should be truthy', (point: unknown) => {
     expect(isPoint(point)).toBeTruthy()
```

Comment:
Why not checking more strictly for it shouldn't to be `true` (like the other type guards are tested).

Reply:
I think I bring it from other test, I will make more strict.